docs/source/api.rst

```diff
@@ -69,3 +105,14 @@ UMAP
 
 .. autoclass:: cuml.UMAP
```

Comment:
sklearn has t-sne under a "manifold" section and UMAP would belong there as well. Their dimensionality reduction algorithms include PCA/SVD/ICA. I believe random projections could be here as well,which is missing from these docs, currently.

Reply:
I'd be happy to keep manifold stuff under dimensionality reduction personally or make it "Dimensionality Reduction and Manifold Learning"? The use cases are pretty closely related

Reply:
Done.

Reply:
It seems like sklearn makes the distinction between nonlinear nearest-neighbors based approaches and linear transformation approaches. 

I'm point this out assuming we are trying to match the way sklearn has categorized the algorithms, but if that's not the goal then I'm okay with putting them together. We should add random projections, though.